src/Embed.js

```diff
@@ -93,15 +93,19 @@ export function embed(config = {}) {
       return element;
     };
 
-    debug('Embeding Configuration', config);
+    debug('Embedding Configuration', config);
 
     // The id for this embedded form.
     const id = `formio-${Math.random().toString(36).substring(7)}`;
     config.id = id;
 
     debug('Creating form wrapper');
-    document.write(`<div id="${id}-wrapper"></div>`);
-    let wrapper = document.getElementById(`${id}-wrapper`);
+    let wrapper = createElement('div', {
```

Comment:
This needs to be ```document.createElement```.  Also, the options to this method will not work with IE, so this should probably be the following.

```
let wrapper = document.createElement('div');
wrapper.id = `${id}-wrapper`;
```